docs/rule/element-attributes-order.md

```diff
@@ -0,0 +1,49 @@
+# element-attributes-order
+
+Element attributes must be ordered, by default the order is:
+
+- arguments
+- attributes
+- element modifiers
+- ...attributes
+- ?attributes
+
+## Configuration
+
+The following values are valid configuration:
+
+- attributeOrder: default [`arguments`, `attributes`, `modifiers`, `splattributes`]
+- alphabetize: default `false`, `true` to enforce alphabetically ordered values within attributes
```

Comment:
What's the reasoning for default `false` for `alphabetize`? Just too noisy for most people?

Reply:
That was my original thought but I think I can follow-up this PR with an auto-fixer which should address this concern.

Reply:
So you would prefer `true` as the default for `alphabetize` given that the autofixer will handle it? If that's the case, we need to use the ideal default now, as changing the default later is a breaking change.

Reply:
Forgot to update the docs, `alphabetize` now defaults to `true`!